libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -334,6 +335,13 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
         this.formFieldChildren.forEach((formField) => this.updateFormFieldProperties(formField));
     }
 
+    /** @hidden */
+    private validateErrorHandler(): void {
```

Comment:
should be prefixed with `_`

Reply:
Please see https://github.com/SAP/fundamental-ngx/issues/4248